src/main/java/uk/gov/hmcts/reform/juddata/camel/route/ParentOrchestrationRoute.java

```diff
@@ -92,64 +93,67 @@ public void startRoute() throws Exception {
-                                .end();
+                            //Archive Blob files
+                            from(archivalRoute)
+                                    .loop(archivalFileNames.size())
+                                    .process(azureFileProcessor)
+                                    .toD(archivalPath + "${header.filename}?" + archivalCred)
+                                    .end();
 
 
-                        for (RouteProperties route : routePropertiesList) {
+                            for (RouteProperties route : routePropertiesList) {
 
-                            Expression exp = new SimpleExpression(route.getBlobPath());
+                                Expression exp = new SimpleExpression(route.getBlobPath());
 
-                            from(DIRECT_ROUTE + route.getRouteName()).id(DIRECT_ROUTE + route.getRouteName())
-                                    .transacted()
-                                    .policy(springTransactionPolicy)
-                                    .setProperty(BLOBPATH, exp)
-                                    .process(fileReadProcessor).unmarshal().bindy(BindyType.Csv,
-                                    applicationContext.getBean(route.getBinder()).getClass())
-                                    .to(route.getTruncateSql())
-                                    .process((Processor) applicationContext.getBean(route.getProcessor()))
-                                    .split().body()
-                                    .streaming()
-                                    .bean(applicationContext.getBean(route.getMapper()), MAPPING_METHOD)
-                                    .to(route.getSql()).end();
-                        }
+                                from(DIRECT_ROUTE + route.getRouteName()).id(DIRECT_ROUTE + route.getRouteName())
+                                        .transacted()
+                                        .policy(springTransactionPolicy)
+                                        .setProperty(BLOBPATH, exp)
+                                        .process(fileReadProcessor).unmarshal().bindy(BindyType.Csv,
+                                        applicationContext.getBean(route.getBinder()).getClass())
+                                        .to(route.getTruncateSql())
+                                        .process((Processor) applicationContext.getBean(route.getProcessor()))
+                                        .split().body()
+                                        .streaming()
+                                        .bean(applicationContext.getBean(route.getMapper()), MAPPING_METHOD)
+                                        .to(route.getSql()).end();
+                            }
 
-                    }
-                });
+                        }
+                    });
+        } catch (Exception ex) {
+            throw new FailedToCreateRouteException("ParentOrchestrationRoute failed to start", startRoute, ex);
```

Comment:
can u put exception message as - ("Judicial Data Load - ParentOrchestrationRoute failed to start", startRoute, ex);

Reply:
Also change anywhere we are putting exception messages to include Judicial Data Load - <
<RouteName>

Reply:
done